flipperdevices/flipperzero-firmware
# Skylanders plugin

# What's new

- NFC Plugin to detect skylanders and show piece name (Need to add more from https://github.com/silicontrip/SkyReader/blob/master/toynames.cpp)

# Verification 

- Load [Chest.nfc.txt](https://github.com/flipperdevices/flipperzero-firmware/files/13760047/Chest.nfc.txt) onto flipper
- Open NFC app
- Select Saved->Chest->Info
- See "Skylanders" and "Imaginite Mystery Chest" (name from https://skylanders.fandom.com/wiki/Imaginite_Mystery_Chests)

# Checklist (For Reviewer)
